frontend/src/shared/constants/healthCheckTypes.ts

```diff
@@ -7,4 +7,54 @@ export const healthCheckTypes = [
     logo: mongoDbLogo,
     category: healthCheckCategories.DATABASE,
   },
+  {
```

Comment:
Please remove duplicated objects here. It should have only one health check type "MongoDB".